osu.Game.Rulesets.Osu/Difficulty/Skills/OsuStrainSkill.cs

```diff
@@ -34,6 +34,9 @@ public abstract class OsuStrainSkill : StrainSkill
         /// </summary>
         protected virtual double DifficultyMultiplier => DEFAULT_DIFFICULTY_MULTIPLIER;
 
+        private const double a = 5.0;
```

Comment:
I don't think those constants should be put here and rather either given a proper name and exist here or just be moved directly where the values are used

Reply:
Yeah that seems reasonable. Can make that change. I just wanted to explicitely point out that there were arbitrary constants being used.